ember-browser-services/src/test-support/window-mock-augments.ts

```diff
@@ -1,56 +0,0 @@
-import window from 'ember-window-mock';
-
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-expect-error
-import locationFactory from 'ember-window-mock/test-support/-private/mock/location';
-
-const AUGMENTS: Array<string | symbol> = ['origin'];
-
-function createLocation(target?: Window) {
-  let initialHref = target?.location?.href ?? window.location.href;
-  let mockLocation = locationFactory(initialHref);
-  let values: any = {};
-
-  mockLocation.isPatchedLocation = true;
-
-  return new Proxy(mockLocation, {
-    get(target, key, receiver) {
-      if (AUGMENTS.includes(key)) {
-        return values[key] ?? Reflect.get(target, key, receiver);
-      }
-
-      return Reflect.get(target, key, receiver);
-    },
-
-    set(target, key, value, receiver) {
-      if (AUGMENTS.includes(key)) {
-        return (values[key] = value);
-      }
-
-      return Reflect.set(target, key, value, receiver);
-    },
-  });
-}
-
-export function patchWindow(target: any) {
-  let location = createLocation(target);
-
-  let self: any = new Proxy(target, {
-    get(target, key, receiver) {
-      if (key === 'location') return location;
-      if (key === 'parent') return self;
-      if (key === 'top') return self;
```

Comment:
This behaviour is now ensured upstream, see https://github.com/simonihmig/ember-window-mock/pull/575/files#diff-cbde1dc33d38be8541d8e3c47e25031991ff2948aca51f1ad25d20a8ecd875e5R38-R40